app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    comm, file, new_file = command.split()
```

Comment:
Ensure that the input string contains exactly three elements separated by spaces and starts with 'cp'. This is mentioned in the checklist: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'